test/resources.test.js

```diff
@@ -167,35 +167,37 @@ describe('Test Resources responses', function () {
     it('Ensure a chain of added filters reduces resultset correctly', function (done) {
```

Comment:
I like the idea of iterating and building `nextUrl`, and in each iteration making sure it's still valid.
It _does_ feel a little fragile that these are magic numbers that are depend on the contents of the ElasticSearch index
at the time we run the tests.

Would this still be a useful test if we found a way to loop over the `dates` and just ensure that in each iteration / that the result set gets smaller than the previous iteration?

I prefer explicit tests more than overly clever tests - and what we have is certainly readable and explicit.
But it would feel nice to get to "Is the result set smaller than it was a second ago" without having to peg the assertions to actual numbers.